### ISSUE TITLE
Vscode swaps theirs/ours on deleted file during rebase

### ISSUE THREAD

**Issue by johnzielke** [microsoft/vscode#113593]
Version: 1.52.1 (system setup)
Commit: ea3859d4ba2f3e577a159bc91e3074c5d85c0523
Date: 2020-12-16T16:34:46.910Z
Electron: 9.3.5
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.19042

Steps to Reproduce:
1. Modify file in local branch and commit
2. Rebase onto master that has that same file deleted
3. Code shows file as "deleted by Us"

![20201230_144509](https://user-images.githubusercontent.com/20091488/103356056-32170100-4ab0-11eb-943b-a12dfd13e44d.jpg)
4. Pycharm shows the file as deleted by "them" which is the way it should be in my opinion
![image](https://user-images.githubusercontent.com/20091488/103356070-39d6a580-4ab0-11eb-8a61-eb98bc4bfb5c.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by eamodio** [comment]
I believe we are honoring the Git "us"/"them" that git uses -- which I agree feels backward with rebase. We also avoid the us/them terminology in other places (using current/incoming) instead, since that IMO aligns better and is less confusing. We should probably adopt that here.

**Comment by johnzielke** [comment]
I agree, while this may be [how git interprets it](https://stackoverflow.com/questions/21025314/who-is-us-and-who-is-them-according-to-git), I think that especially for beginners in git, this is counterintuitive and might lead to confusion.

**Comment by Kurt-von-Laven** [comment]
Indeed! I greatly prefer the current/incoming nomenclature VSCode uses, and it would be great to use that consistently.
